distribution/zip/ballerina/bin/ballerina

```diff
@@ -136,25 +136,17 @@ fi
 
 # ----- Process the input command ----------------------------------------------
 
-for c in "$@"
-do
-    if [ "$c" = "--java.debug" ] || [ "$c" = "-java.debug" ] || [ "$c" = "java.debug" ]; then
-          CMD="--java.debug"
-    elif [ "$CMD" = "--java.debug" ] && [ -z "$PORT" ]; then
-          PORT=$c
+if [ -n "${BAL_JAVA_DEBUG+set}" ]; then
```

Comment:
Why do we need `+set`?

Reply:
This is to distinguish between `BAL_JAVA_DEBUG` not being set and `BAL_JAVA_DEBUG` being set to an empty string/nothing [1].

For example if it is set to an empty string we want to say "Please specify the debug port for the BAL_JAVA_DEBUG variable"

[1] https://serverfault.com/a/382740